src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -189,7 +189,10 @@ protected function describeContainerDefinition(Definition $definition, array $op
         $output .= "\n".'- Abstract: '.($definition->isAbstract() ? 'yes' : 'no');
 
         if (method_exists($definition, 'isAutowired')) {
```

Comment:
@dunglas : Do you remember why this check was needed?
To me it looks weird, because it was introduced in 2.8, where the FrameworkBundle already required `symfony/dependency-injection: "~2.8"`, so we can't get a `Definition` instance not having this method.

Maybe I'm missing something, but it looks safe to remove.